DAQ/src/StrawRecoFromFragments_module.cc

```diff
@@ -129,8 +133,6 @@ void art::StrawRecoFromFragmnets::produce(Event& event) {
   if (numTrkFrags == 0) {
     std::cout << "[StrawRecoFromFragmnets::produce] found no Tracker fragments!"
 	      << std::endl;
-    event.put(std::move(straw_digis));
```

Comment:
This comment is out of the scope of the review.

For jobs running offline ( ie not in the trigger) we have a policy that we will produce empty data products in this case.  So it is an error for down stream code to request a data product and not find it.  

I can understand that for modules running in the trigger there could be a different policy because of the overhead of creating data products.     I don't remember a discussion about this but there was  a period when I did not routinely attend TDAQ meetings.